draft-wood-ohai-unreliable-ohttp.md

```diff
@@ -227,6 +247,21 @@ the choice of how to buffer incoming requests is an implementation decision, and
 any delay in processing a request can increase the likelihood that the request
 is dropped or discarded due to it being stale.
 
+If an Oblivious Gateway Resource requires unreliable delivery for a
+request, by implementation constraint or policy, it SHOULD respond
+with `406 Not Acceptable` to any requests which require return of an
+Encapsulated Response. The Relay Resource will forward this to the
+Client, which can fail or retry with unreliable delivery according to
+its own requirements.
+
+Note that while it's possible for the Gateway to decide whether to
```

Comment:
This is interesting =) Given the privacy implications here, I would recommend we remove this text and file an issue to consider adding it back.

Reply:
It might make more sense in the Security section, but only after we add the `Accept: message/*` bit. I will move.

Reply:
https://github.com/chris-wood/draft-unreliable-ohttp/issues/3